go.mod

```diff
@@ -18,8 +18,7 @@ require (
 	github.com/docker/go-units v0.4.0
 	github.com/go-sql-driver/mysql v1.6.0
 	github.com/gofrs/uuid v4.2.0+incompatible
-	github.com/golang-collections/collections v0.0.0-20130729185459-604e922904d3
-	github.com/golang/glog v1.1.0
+	github.com/golang/glog v1.2.4
```

Comment:
Hmm… I can't reopen that comment, so I'll try again, if you merge, you will not see that change in your PR.

After
```
git merge origin/master
```
I see the following different merge-bases, showing that your fork is not merged with the latest.

```
(oath_tools) [scr@MacBook-Pro-3]$ git merge-base origin/master pixfuture-media/master
243c0f06f0f9ea2cb6cf0994dc7791ae63e957ed
(oath_tools) [scr@MacBook-Pro-3]$ git merge-base origin/master HEAD
f2d62efa4f0336185a22fff1ff31bab5cea53162
```